be/src/runtime/mem_tracker.h

```diff
@@ -157,6 +157,7 @@ class MemTracker {
     WARN_UNUSED_RESULT
     MemTracker* try_consume(int64_t bytes) {
         if (UNLIKELY(bytes <= 0)) return nullptr;
+        _last_try_consume = bytes;
```

Comment:
If multiple threads are bound to a MemTracker, when printing the log, the memory allocated for the last time may not be the actual memory to be allocated.

Reply:
Probably using thread local variables is a better option。

Reply:
+1